drivers/periph_common/i2c.c

```diff
@@ -37,7 +37,7 @@ int i2c_read_regs(i2c_t dev, uint16_t addr, uint16_t reg,
                   void *data, size_t len, uint8_t flags)
 {
     /* First set ADDR and register with no stop */
-    int8_t ret = i2c_write_bytes(dev, addr, &reg, (flags & I2C_REG16) ? 2 : 1,
+    int ret = i2c_write_bytes(dev, addr, &reg, (flags & I2C_REG16) ? 2 : 1,
```

Comment:
The alignment with the second line is incorrect now. Same below

Reply:
Should be good now